### DISCUSSION TITLE
Repository names have tooltips that shrink the available click target

### DISCUSSION THREAD

**Discussion by peterldowns** [community/community#114049] [Repositories] [answered]
### Select Topic Area

Bug

### Body

I recorded this in Firefox on macOS but I was able to reproduce in both Safari and Chrome as well, so most likely it's a cross-browser problem. I'm not sure when this got added, but repositories now have tooltips that simply repeat the name of the repository (why?) and obscure the click-target of the underlying link to the repository. This makes click on the repository links too difficult and it's really annoying. My guess is this is a mistake.

https://github.com/community/community/assets/824173/782e753e-2730-4a03-8762-5dfe6efa0b88

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @peterldowns! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Comment by peterldowns** [comment, marked as answer]
@ghostinhershell this is an accessibility problem with the product — the popovers make the "target size" of the link smaller than expected and difficult to click. This difficulty affects everyone, but is particularly noticeable for people with fine motor disabilities that prevent them from accurately targeting such a small area. This is often discussed in the context of ["touch targets"](https://web.dev/articles/accessible-tap-targets) but it applies equally to all input devices, including pointers/mousing.

Thankfully, this bug now seems to be fixed, and I can no longer reproduce the problem.
